src/RazorSdk/Tool/Microsoft.NET.Sdk.Razor.Tool.csproj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Razor is a markup syntax for adding server-side logic to web pages. This assembly contains infrastructure supporting Razor MSBuild integration.</Description>
+    <TargetFramework>$(SdkTargetFramework)</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <AssemblyName>rzc</AssemblyName>
+    <StrongNameKeyId>MicrosoftAspNetCore</StrongNameKeyId>
+
+    <!-- This is not a package, it is part of Razor SDK. -->
+    <IsPackable>false</IsPackable>
+    <IsShipping>false</IsShipping>
+
+    <!-- Don't produce rzc.exe -->
+    <UseAppHost>false</UseAppHost>
+    <RuntimeIdentifier />
+
+    <!-- Need to build this project in source build -->
+    <ExcludeFromSourceBuild>false</ExcludeFromSourceBuild>
```

Comment:
Is this a thing in this repo? I was under the impression `ExcludeFromSourceBuild` is an aspnetcore thing.